indico/modules/rb/schemas.py

```diff
@@ -193,13 +196,14 @@ class Meta:
 
 class ReservationOccurrenceSchema(mm.SQLAlchemyAutoSchema):
     reservation = Nested(ReservationSchema)
+    link = Nested(ReservationOccurrenceLinkSchema)
```

Comment:
Did you verify hat this does not cause query spam (`O(n)` queries for each booking when viewing the calendar with lots of bookings)?

Reply:
You have to test it (the `db_log.py` script is great for this purpose). For APIs that return a large amount of data we usually specify exactly what we want to query/join so if you suddenly access something new (that's actually a relationship and not just a column), then there's good chance you need to add a `joinedload('link')` to the query options.

Reply:
Just added the `joinedload`.